plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/crysl/templates/encryptionhybridfiles/SecureEncryptor.java

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2015-2019 TU Darmstadt, Paderborn University
+ * Copyright (c) 2015-2021 TU Darmstadt, Paderborn University

```

Comment:
Same thing here as well. I am not finding the right combination of options in the wizard in order to generate this template and test its generation. Thanks if you could help me.

Reply:
In this case, answer the first question, Unencrypted digital channel (e.g., email) and the second one, file.